### PR TITLE
Bump to `0.11.0-alpha.1+dev`

### PR DESCRIPTION
**Pull request by jprochazk**
### Checklist
* [x] I have read and agree to [Contributor Guide](https://github.com/rerun-io/rerun/blob/main/CONTRIBUTING.md) and the [Code of Conduct](https://github.com/rerun-io/rerun/blob/main/CODE_OF_CONDUCT.md)
* [x] I've included a screenshot or gif (if applicable)
* [x] I have tested [demo.rerun.io](https://demo.rerun.io/pr/4114) (if applicable)
* [x] The PR title and labels are set such as to maximize their usefulness for the next release's CHANGELOG

- [PR Build Summary](https://build.rerun.io/pr/4114)
- [Docs preview](https://rerun.io/preview/c739784e386f8374ec1daf64050de966033b3969/docs) 
- [Examples preview](https://rerun.io/preview/c739784e386f8374ec1daf64050de966033b3969/examples) 
- [Recent benchmark results](https://ref.rerun.io/dev/bench/)
- [Wasm size tracking](https://ref.rerun.io/dev/sizes/)

### CODE REVIEW

File: Cargo.toml [hunk 1]
```diff
@@ -18,56 +18,56 @@ include = ["../../LICENSE-APACHE", "../../LICENSE-MIT", "**/*.rs", "Cargo.toml"]
 license = "MIT OR Apache-2.0"
 repository = "https://github.com/rerun-io/rerun"
 rust-version = "1.72"
-version = "0.10.0"
+version = "0.11.0-alpha.1+dev"
```
Comment (jleibs) [inline Cargo.toml:21]: Can you remind me what the +dev does here?

**Review by jleibs — APPROVED** [review]

File: Cargo.toml (hunk 1 above)
Comment (emilk) [reply Cargo.toml:21]: It means "not-yet-released" - it is not the published `alpha.1`, but it is building towards it

Comment (jprochazk) [reply Cargo.toml:21]: That means it's an unreleased version, so it's clearly marked as such in the various builds we do both locally and on CI.

The versioning scheme is supposed to be `0.10.0` -> `0.11.0-alpha.1+dev` -> `0.11.0-alpha.1` -> `0.11.0-alpha.2+dev` -> `0.11.0-rc.1` -> `0.11.0` -> `0.12.0-alpha.1+dev` -> etc.

Comment (jleibs) [reply Cargo.toml:21]: Makes sense! Thanks.

**Pull request merged**